src/QueueReceiver.Worker/QueueReceiver.Worker.csproj

```diff
@@ -8,15 +8,9 @@
     <OutputType>Exe</OutputType>
     <StartupObject />
     <Nullable>enable</Nullable>
+    <IsPackable>true</IsPackable>
```

Comment:
Does this need to be specified when true is default?

Reply:
yep, doesn't pack unless its added. Source: 2 hours of debugging